articles/policies/rate-limits-mgmt-api.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Management API Endpoint Rate Limits
+description: Describes Auth0's rate limit policy when working with Auth0 Management API endpoints.
+toc: true
+topics:
+    - auth0-policies
+    - rate-limits
+contentType:
+  - reference
+useCase:
+  - support
+---
+# Management API Endpoint Rate Limits
+
+::: warning
+These rate limits apply to you if you subscribed on or after **May 19, 2020**. If you subscribed before **May 19, 2020**, the older [rate limit policy](/policies/legacy-rate-limits) applies to you until **June 18, 2020**. Starting on **June 18, 2020**, these limits will apply to all tenants. You will be notified of the new limits through a **Dashboard Notification**. If the changes will impact your tenant, you will be notified directly via email with additional information about minimizing API calls and upgrading plans.
+:::
+
+The rate limits for this API differ depending on whether your tenant is free or paid, production or not.
+
+| Tenant Type | Rate Limit (per second) | Rate Limit (per minute) |
+| - | - | - |
+| Free or Trial | 10 | 120 |
+| Developer or Developer Pro (created before May 19, 2020) | 50 | 1000 |
+| Enterprise (Production) | 50 | 1000 |
+| Enterprise (Non-production) | 10 | 120 |
+
+The rate limits include calls made via [Rules](/rules) and are set **by tenant** and not by endpoint.
+
+Each endpoint is configured with a bucket that defines:
+
+-  Request limit
+-  Rate limit window (per second, per minute, per day, etc.)
+
+```text
+bucket:
+    size: x
+    per_minute: y
+```
+
+For example, the above states that, for the given bucket, there is a maximum request limit of `x` per minute, and for each minute that elapses, permissions for `y` requests are added back. In other words, for each `60 / y` seconds, one additional request is added to the bucket. This occurs automatically until the bucket contains the maximum permitted number of requests.
+
+For some API endpoints, the rate limits are defined per bucket, so the origins of the call do not influence the rate limit changes. For other buckets, the rate limits are defined using different keys, so the originating IP address is considered when counting the number of received API calls.
+
+::: note
+If you are using an API endpoint **not** listed below and you receive rate limit headers as part of your response, see [Anomaly Detection](/anomaly-detection) for more information.
+:::
+
+The following Auth0 Management API endpoints return rate limit-related headers. For additional information about these endpoints, please consult the [Management API explorer](/api/management/v2).
+
+## Developer and Developer Pro subscription limits
+
+| Endpoint | Path | Rate Limit (per second) | Rate Limit (per minute) |
+| - | - | - | - |
+| Read users | `GET /api/v2/users` | 40 | 500 |
+| Write users | `POST /api/v2/users` | 20 | 200 |
```

Comment:
What we can do here? Is it no a single endpoint, but multiple paths for each group....
This is also dynamic, we can add more endpoints and do not want to miss docs updates.
Maybe we can just add one example per group?

Reply:
I see what you're saying. I think you mentioned this in a previously resolved comment. Do we need the actual endpoint path at all? Should I just review that whole column maybe? I'm not sure I can decide. We might need a third-opinion.

Reply:
do you think we need to add all paths for each group there? If not, please remove the column.
e.g: Read users
GET /api/v2/users
GET /api/v2/users-by-email
GET /api/v2/users/{id}